soh/soh/Enhancements/randomizer/randomizer_check_tracker.cpp

```diff
@@ -1384,7 +1384,9 @@ void UpdateAllAreas() {
 }
 
 void UpdateAreas(RandomizerCheckArea area) {
-    areasFullyChecked[area] = areaChecksGotten[area] == checksByArea.find(area)->second.size();
+    if (checksByArea.contains(area)) {
+        areasFullyChecked[area] = areaChecksGotten[area] == checksByArea.find(area)->second.size();
+    }
```

Comment:
This was changed because it was referencing data that no longer existed. According to Malk this may have been the cause for a remaining save corruption glitch (occuring most frequently on console), but I was running into it 100% of the time with the soft reset saving.